packages/shared/src/util/features.ts

```diff
@@ -744,7 +744,7 @@ export function evalDeterministicPrereqValue(
   const parsedCondition = getParsedPrereqCondition(condition);
   if (!parsedCondition) return "fail";
   const evalObj = { value: value };
-  const pass = evalCondition(evalObj, parsedCondition);
+  const pass = evalCondition(evalObj, parsedCondition, {});
```

Comment:
save for a different PR? but is this safe to pass {} for context?

Reply:
Good point; this import is version locked so it won't get the new signature right away huh. I think I just changed it here bc I'd yarn linked the package and it was complaining about missing an arg

Reply:
i guess it goes back to whether that arg should be optional or not. If not, then you probably need to do both at the same time.